atomic_reactor/plugins/post_fetch_docker_archive.py

```diff
@@ -25,6 +24,20 @@ def __init__(self, workflow):
         super(FetchDockerArchivePlugin, self).__init__(workflow)
         self.source_build = bool(self.workflow.data.build_result.source_docker_archive)
 
+    def download_image(self, build_dir: BuildDir):
+        image = self.workflow.data.tag_conf.get_unique_images_with_platform(build_dir.platform)[0]
+        image_path = os.path.join(build_dir.path, EXPORTED_SQUASHED_IMAGE_NAME)
+        image_type = IMAGE_TYPE_DOCKER_ARCHIVE
+
+        self.log.info('fetching image %s', image)
+        self.workflow.imageutil.download_image_archive_tarball(image, image_path)
+
+        metadata = get_exported_image_metadata(image_path, image_type)
+
+        self.workflow.data.exported_image_sequence.append(metadata)
```

Comment:
It doesn't seem right to put the 4 arch-specific images into the `exported_image_sequence` - plugins assume that everything in this sequence is the same image (don't ask me why it is a sequence then :smile:).

IMO this plugin should not update `exported_image_sequence` anymore. All the plugins that currently depend on the sequence -- appears to be `tag_and_push`, `store_metadata` and `koji_import` (through `utils.koji.get_output`) -- will have to be reworked for multiple platforms and, as part of that, they will have to stop using this attr.

It might be good to add `OSBS2 TBD` into the places where the sequence is currently used, that it should not be used anymore

Reply:
Maybe, instead of appending to the sequence, we could just `return` the metadata. That means `for_each_platform()` would return a dict of platform=>metadata, which we could return from the plugin

Reply:
added `OSBS2 TBD` and updated the plugin to return platform=>metadata